sources/packages/api/src-sql/UsersDraft/Create-UsersDraft.sql

```diff
@@ -0,0 +1,28 @@
+-- User Draft Table
+CREATE TABLE IF NOT EXISTS users_draft (
+  id SERIAL PRIMARY KEY,
+  -- Reference column
+  user_name VARCHAR(300) NOT NULL UNIQUE REFERENCES users(user_name) ON DELETE CASCADE,
+  -- Data
+  data JSONB NOT NULL,
+
+  -- Form path
+  form_path VARCHAR(100) NOT NULL,
```

Comment:
I believe that, if we are going to use this generic table to store applications, for instance, the same user could have more than one draft at the same time. We need to check this possibility and maybe adjust accordingly.

Reply:
Here, my design decision was one draft per unique form. But yes there is a huge scope of improvement.

Reply:
I totally agree. We do not have a solid requirement yet. In general, I liked the design 👍